nginx/proxy_ssl.conf

```diff
@@ -2,14 +2,25 @@ upstream target_service {
   server {{TARGET_SERVICE}};
 }
 
+upstream cert_service {
+  server {{CERT_SERVICE}};
```

Comment:
Shouldn't this be an optional setting? Currently, if you don't pass in a `CERT_SERVICE` you can't start nginx:

```
$ docker run -e TARGET_SERVICE=localhost -e ENABLE_SSL=true 02c301a264d3
Enabling SSL...
Starting nginx...
2015/12/15 15:22:22 [emerg] 10#10: invalid number of arguments in "server" directive in /etc/nginx/conf.d/proxy.conf:6
nginx: [emerg] invalid number of arguments in "server" directive in /etc/nginx/conf.d/proxy.conf:6
```